service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -221,6 +221,15 @@ public boolean isSendingNoticeOfProceedings() {
     private List<Element<HearingBooking>> cancelledHearingDetails;
     private final List<Element<UUID>> selectedHearingIds;
 
+    @JsonIgnore
+    public HearingBooking getHearingBookingByUUID(UUID elementId) {
+        return hearingDetails.stream()
```

Comment:
hearingDetails can be null,
you can use:
ElementUtils.findElement(elementId, hearingDetails).orElse(null)